apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -0,0 +1,696 @@
+
+  useEffect(() => {
+    if (urlCategory) {
+      const validCategory = CATEGORY_OPTIONS.find((option) => {
+        if (option.value.toLowerCase() === ((urlCategory as string) ?? '').toLowerCase())
+          return option
+      })
+      if (validCategory !== undefined) form.setValue('category', validCategory.value)
+    }
+  }, [urlCategory])
+
+  useEffect(() => {
+    if (urlSubject) form.setValue('subject', urlSubject)
+  }, [urlSubject])
+
+  useEffect(() => {
+    if (urlMessage) form.setValue('message', urlMessage)
+  }, [urlMessage])
+
+  return (
+    <Form_Shadcn_ {...form}>
+      <form
+        id="support-form"
+        className={cn('flex flex-col gap-y-8 max-w-[620px]')}
+        onSubmit={form.handleSubmit(onSubmit)}
+      >
+        <h3 className="text-xl px-6">How can we help?</h3>
+
+        <FormField_Shadcn_
+          name="organizationSlug"
+          control={form.control}
+          render={({ field }) => (
+            <FormItemLayout
+              className="px-6"
+              layout="vertical"
+              label="Which organization is affected?"
+            >
+              <FormControl_Shadcn_>
+                <Select_Shadcn_
+                  {...field}
+                  defaultValue={field.value}
+                  onValueChange={field.onChange}
+                >
+                  <SelectTrigger_Shadcn_ className="w-full">
+                    <SelectValue_Shadcn_ asChild placeholder="Select an organization">
+                      <div className="flex items-center gap-x-2">
+                        {(organizations ?? []).find((o) => o.slug === field.value)?.name}
+                        {isLoadingSubscription && <Loader2 size={14} className="animate-spin" />}
+                        {isSuccessSubscription && (
+                          <Badge variant="outline" className="capitalize">
+                            {subscriptionPlanId}
+                          </Badge>
+                        )}
+                      </div>
+                    </SelectValue_Shadcn_>
+                  </SelectTrigger_Shadcn_>
+                  <SelectContent_Shadcn_>
+                    <SelectGroup_Shadcn_>
+                      {organizations?.map((org) => (
+                        <SelectItem_Shadcn_ value={org.slug}>{org.name}</SelectItem_Shadcn_>
+                      ))}
+                    </SelectGroup_Shadcn_>
+                  </SelectContent_Shadcn_>
+                </Select_Shadcn_>
+              </FormControl_Shadcn_>
+            </FormItemLayout>
+          )}
+        />
+
+        <div className="flex flex-col gap-y-2 px-6">
+          <FormField_Shadcn_
+            name="projectRef"
+            control={form.control}
+            render={({ field }) => (
+              <FormItemLayout layout="vertical" label="Which project is affected?">
+                <FormControl_Shadcn_>
+                  <Select_Shadcn_
+                    {...field}
```

Comment:
don't think you need the spread operator

Reply:
hmm i think we might need it actually - without the spread operator, the initial form.setValue from the useEffect doesnt update this field